### ISSUE TITLE
`update-xfce-settings` is no longer tested in CI

### ISSUE THREAD

**Issue by legoktm** [freedomofpress/securedrop-workstation#1062]
* [x] I have searched for duplicates or related issues

## Description

An unintentional regression of https://github.com/freedomofpress/securedrop-workstation/commit/539ea5a32aae680a3f3338d8339498be4b560b21 is that the systemd units are triggered on boot and not during provisioning, so we don't test them in CI.

Note that this was flaky in the past (see #1006), so some care might be needed.

**Comment by rocodes** [comment]
Yeah, there are a few things that have moved to being provisioned on boot and not during provisioning, which isn't ideal from a testing point of view but is maybe better from an 'enforcing system state every boot time' pov:

- xfce adjust icon size (all build types)
- xfce power settings (staging, prod)
- logind override (staging, prod)

and there are hopefully/eventually some more to come (like the sdw-notify timer and service).

I'm not sure the best way to handle: a) the fact that there're some configurations we don't test if we only have dev CI, and b) the stuff that requires reboot. (Should we reboot then run dom0 tests? is that possible?)
